internal/x/registry.go

```diff
@@ -12,3 +15,7 @@ type LoggerProvider interface {
 type WriterProvider interface {
 	Writer() herodot.Writer
 }
+
+type TracingProvider interface {
```

Comment:
Small nit, would `TracerProvider` fit better to the existing provider naming scheme?

Reply:
Ideally yes, but we've stuck to `TracingProvider` across the board everywhere else (in Backoffice, Kratos, ...) so I suppose we can leave it as is for consistency.

Reply:
Ah thanks, makes sense